lib/slim-rails/register_engine.rb

```diff
@@ -0,0 +1,30 @@
+module Slim
+  module Rails
+    module RegisterEngine
+      def self.register_engine(app, config)
+        if ::Rails::VERSION::MAJOR == 3
+          register_engine3(app, config)
+        else
+          register_engine4(app, config)
+        end
+      end
+
+      private
+
+      def self.register_engine3(app, _config)
+        return unless app.assets
+        return unless app.assets.respond_to?(:register_engine)
+
+        app.assets.register_engine('.slim', Slim::Template)
+      end
+
+      def self.register_engine4(_app, config)
```

Comment:
If changes the parameter for each Rails version, register_engine will handle which `app` or `config` is needed for each Rails version. Is it ok?